### PR TITLE
Fix the test when a benefactor Withdraws sometime after a token is foreclosed before foreclosure is enforced on chain.  

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: test/steward_benefactor_withdrawal.js [hunk 1]
```diff
@@ -126,34 +131,34 @@ contract("WildcardSteward Benefactor collection", (accounts) => {
...
+    });
+    // price * (now - timeLastCollected) * patronageNumerator/ patronageDenominator / 365 days;
+    const due = tokenPrice
+      .mul(time.duration.days(365))
+      .mul(new BN(patronageNumerator))
+      .div(new BN(patronageDenominator))
+      .div(time.duration.days(365));
+
+    assert.equal((await balTrack.delta()).toString(), due.toString());
+  });
```
Comment (JasoonS) [inline test/steward_benefactor_withdrawal.js:161]: Ignore this whole section, I just uncommented it. I made zero changes here.

File: test/steward_benefactor_withdrawal.js [hunk 2]
```diff
@@ -280,6 +280,21 @@ contract("WildcardSteward Benefactor collection", (accounts) => {
...
       );
+      assert.equal(
+        benefactorFundsBefore.toString(),
+        "0",
+        "benefactor should start with a balance of zero"
+      );
+      assert.equal(
+        patronScaledCostBefore.toString(),
+        tokenPrice.mul(new BN(patronageNumerator)).toString(),
+        "benefactor funds should stay zero"
```
Comment (moose-code) [inline test/steward_benefactor_withdrawal.js:291]: Wrong reason in this string if assert fails. Duplicated from below assert

Comment (JasoonS) [inline test/steward_benefactor_withdrawal.js:291]:
```suggestion
        "The scaled token price is incorrect before the withdrawal"
```

**Comment by JasoonS** [comment]
@moose-code merging this

**Pull request merged**